Breadth_First_Search/BFS_in_Csharp.cs

```diff
@@ -0,0 +1,128 @@
+       {                                                 
+           adj[c].Add(v);                                
+           adj[v].Add(c);                                
+       }                                                  
+       public void BFS(int v)                          // function for Breadth First Search ,starting node as parameter
+       {
+           bool[] visited=new bool[vertices];          // visited array 
+           Queue<Int32> queue =new Queue<int>();       // queue declared
+           visited[v]=true;                            
+           queue.Enqueue(v);                           // enqueuing first node in queue
+           while(queue.Count!=0)
+           {
+               v=queue.Dequeue();                      // dequeue node from beginning of queue
+               Console.Write(v + " ");
+               foreach(int i in adj[v])
+               {
+                   if(!visited[i])                    //only unvisited node pushed into queue
+                   {
+                       visited[i]=true;               //marked visited
+                       queue.Enqueue(i);              //pushed into queue
+                   }
+               }
+           }
+       }
+
+    }
+    class program                                   
+    {
+        static void Main(string[] args)
+        {   
+            Console.WriteLine("Enter number of vertices in a graph");     //Taking input number of vertices
+            int v = Convert.ToInt32(Console.ReadLine());    
+
+            graph g=new graph(v);  
+
+            Console.WriteLine("Enter number of edge in a graph");        //Taking input number of edges
+            int e = Convert.ToInt32(Console.ReadLine());  
+
+            for(int i=0;i<e;i++)                                         
+            {   
+                Console.WriteLine("Enter nodes of " + (i+1) +" edge");     //taking input of edges
+                string s = Console.ReadLine();
+
+                int e1 =Convert.ToInt32(s.Split(" ")[0]);                  //first node 
+                int e2 = Convert.ToInt32(s.Split(" ")[1]);                 //second node
+
+                g.addedge(e1,e2);                                          //adding edge
+            }   
+
+            Console.WriteLine("Breadth First Search of Graph :-");
+            g.BFS(0);                                                       //calling BFS function with starting node provided.
+
+        }
+    }
+}
+
+
+/*Sample Input and Output
+
+
+Enter number of vertices in a graph
+10
+Enter number of edge in a graph
+12
+Enter nodes of 1 edge
+0 1
+Enter nodes of 2 edge
+0 2
+Enter nodes of 3 edge
+1 3
+Enter nodes of 4 edge
+2 4
+Enter nodes of 5 edge
+2 5
+Enter nodes of 6 edge
+3 6
+Enter nodes of 7 edge
+6 4
+Enter nodes of 8 edge
+4 5
+Enter nodes of 9 edge
+4 7
+Enter nodes of 10 edge
+7 8
+Enter nodes of 11 edge
+7 9
+Enter nodes of 12 edge
+8 9
+Breadth First Search of Graph :-
+0 1 2 3 4 5 6 7 8 9
+
```

Comment:
One blank line is enough

Reply:
@singh-shreya6 Extra blank lines removed